blueprints/app/files/app/deprecation-workflow.ts

```diff
@@ -0,0 +1,25 @@
+import setupDeprecationWorkflow from 'ember-cli-deprecation-workflow';
```

Comment:
This will trigger TS errors I assume? Made https://github.com/ember-cli/ember-cli-deprecation-workflow/issues/199.

Reply:
Ope Why does the ember-cli's test suite not catch stuff like this  🙈

Reply:
Approved!

Reply:
I have a PR open that wil catch this.